components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dto/OAuthConsumerAppDTO.java

```diff
@@ -236,15 +236,11 @@ public String getTokenType() {
     public void setTokenType(String tokenType) {
         this.tokenType = tokenType;
     }
-
+    
     public boolean isBypassClientCredentials() {
         return bypassClientCredentials;
     }
 
-    public boolean getBypassClientCredentials() {
```

Comment:
Shall we not remove this method? Instead, deprecate this and instruct to use isBypassClientCredentials.

Reply:
Fixed